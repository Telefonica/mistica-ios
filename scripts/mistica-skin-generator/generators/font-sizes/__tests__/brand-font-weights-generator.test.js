import fs from 'fs';
import {resolve} from 'path';
import {generateBrandFontSizes} from '../brand-font-sizes-generator.js';

test('generateBrandFontSizes', () => {
    const anyBrandTokensPath = resolve(__dirname, 'any-brand-tokens.json');
    const anyBrandTokens = JSON.parse(fs.readFileSync(anyBrandTokensPath, 'utf-8'));
    const brand = {prefix: 'O2New'};
    const result = generateBrandFontSizes(brand, anyBrandTokens);

    expect(result).toMatchSnapshot();
});
