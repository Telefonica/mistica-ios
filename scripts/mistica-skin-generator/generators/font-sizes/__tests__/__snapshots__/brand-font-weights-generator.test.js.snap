// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`generateBrandFontSizes 1`] = `
"
// Generated using Make
// DO NOT EDIT

import Foundation

struct O2NewFontSizes: MisticaFontSizes {
    public var tabsLabel: CGFloat = 16
    public var title2: CGFloat = 20
}
"
`;
